lib/tasks/fake_user_data.rake.example

```diff
@@ -0,0 +1,33 @@
+# Replace <your gmail name> with your GMail name below.
+
+namespace :db do
+  task :fake_user_data do
```

Comment:
Have you tested this script? I'm pretty sure you need `=> :environment` in order for the calls to Community, Household, etc. to work...